tests/test-cases/Group24-Multi-VCH/24-02-Multi-VCH-Delete.robot

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 24-02 - Verify vic-machine delete only removes own cVMs
+Resource  ../../resources/Util.robot
+Suite Teardown  Cleanup VCHs
```

Comment:
this needs to be restructured to avoid trying to cleanup VCH that has already been cleaned up via line 55, you really want a full cleanup only in the case the test fails, i.e.
Test Teardown  Run Keyword If Test Failed  Cleanup VCHs
and in the happy case you need to cleanup the 2nd VCH at the bottom of the test case

Reply:
Done - https://github.com/vmware/vic/pull/7686